content/documentation/validations/index.adoc

```diff
@@ -317,20 +328,13 @@ https://istio.io/docs/tasks/security/authn-policy/#globally-enabling-istio-mutua
 
 '''
 
-[#virtualservices]
-:numbered:
-== VirtualServices
-
-=== DestinationWeight on route doesn't have a valid service (host not found)
-:numbered!:
-
-VirtualService routes matching requests to a service inside your mesh. Routing can also match a subset of traffic to a certain version of it for example. Any service inside the mesh must be targeted by its name, the IP address are only allowed for hosts defined through a Gateway. Host must be in a short name or FQDN format. Short name will evaluate to VS' namespace, regardless of where the actual service might be placed.
+=== KIA0209 - ServiceMeshPolicy enabling mTLS is missing
 
-If the host is not found, Istio ignores the defined rules. However, if a subset with a Destination Rule is not found it affects all the subsets and all the routings. As such, care must be taken that the Destination rule is available before deploying the Virtual Service.
+Maistra has the ability to define mTLS communications at mesh level. In order to do that, Istio needs one DestinationRule and one ServiceMeshPolicy. The DestinationRule configures all the clients of the mesh to use mTLS protocol on their connections. The ServiceMeshPolicy defines what authentication methods can be accepted on the workload of the whole mesh.
+If the ServiceMeshPolicy is not found or doesn't exist and the mesh-wide DestinationRule is on ISTIO_MUTUAL mode, all the communication returns 500 errors.
```

Comment:
"ServiceMeshPolicy is not found at control plane namespace" perhaps better to differentiate.
But just a comment, you can leave it as it is.